### DISCUSSION TITLE
2022/10/16/moving-from-disqus-to-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [bric3/bric3.github.io#46] [Blog Comments]
# 2022/10/16/moving-from-disqus-to-giscus/

Java mostly, and general tech

https://blog.arkey.fr/2022/10/16/moving-from-disqus-to-giscus/

**Comment by propellamstudios** [comment]
Hi, just wanted to let you know that I really found this blog post very useful when adding Giscus to my own blog. Thank you for publishing this post!

**Comment by kauegatto** [comment]
Thanks for the post :)

**Comment by RobinChen121** [comment]
test
